### DISCUSSION TITLE
Add auto-updater

### DISCUSSION THREAD

**Discussion by GingerAdonis** [OpenRCT2/OpenRCT2#15673] [Feature requests]
It would be great to have support for (auto-)updating ORCT2 from the game itself.
- Should be able to switch between branches.
- Can use the .org API for getting latest build.
- Support all available platforms, and for Windows both the installer & zip builds.

Thoughts?

**Comment by Krutonium** [comment]
That would solve my issues, and ideally we could add a content store as well - it was planned and I think someone even wrote the code for it at some point.

**Comment by GingerAdonis** [comment]
I was working on this feature a few months ago. But I was unable to finish, and got rid of the code. (for the updating that is)

**Comment by Gymnasiast** [comment]
@PFCKrutonium : That's correct, yes.

Anyway, I think it's a good idea, although in most Linux distributions setting up a repository to update OpenRCT2 with the system updater will probably work better. So having a compile option to disable the mechanism would be a good thing.

**Comment by RaymondBakker** [comment]
If you're going to add this please make a parameter to turn autoupdated off if desired. (Though enable it by default of course)

**Comment by J-PIE-314** [comment]
The title screen of OpenRCT2 now shows a notice if there is a new release version available.
You may also be interested in using this: https://github.com/IntelOrca/OpenLauncher
